test/transpile/plugins/moduleResolution.ts

```diff
@@ -0,0 +1,12 @@
+import type * as tstl from "../../../src";
+
+const plugin: tstl.Plugin = {
+    moduleResolution(moduleIdentifier, requiringFile) {
+        if (moduleIdentifier.includes("foo")) {
+            return requiringFile.replace("foo", "bar");
```

Comment:
the requiring file is called main.ts (or .lua?), not foo, so I'm confused by this plugin. See also my other comment regarding the requirintFile parameter